tests/core.py

```diff
@@ -602,6 +602,17 @@ def test_scheduler_job(self):
                                 **self.default_scheduler_args)
         job.run()
 
+        log_base_directory = configuration.conf.get("scheduler",
+            "child_process_log_directory")
+        latest_log_directory_path = os.path.join(log_base_directory, "latest")
+        # verify that the symlink to the latest logs exists
+        assert os.path.islink(latest_log_directory_path)
+
+        # verify that the symlink points to the correct log directory
+        log_directory = os.path.join(log_base_directory,
+            datetime.now().strftime("%Y-%m-%d"))
```

Comment:
Could be a race condition here on a day boundary, let's inject a mock time like is done elsewhere in the tests.